src/tiles/grid/item-tile.ts

```diff
@@ -75,10 +76,14 @@ export class ItemTile extends LitElement {
   /**
    * Templates
    */
-  private get doesSortedByDate() {
-    return ['date', 'reviewdate', 'addeddate', 'publicdate'].includes(
-      this.sortParam?.field as string
-    );
+  private get creatorTemplate(): TemplateResult | typeof nothing {
+    if (!this.model?.creator) return nothing;
+
+    return html`
+      <div class="created-by">
+        <span class="truncated">by&nbsp;${this.model?.creator}</span>
```

Comment:
light suggestion: maybe we add `title` attribute here to distinguish what the data means `title='creator'`?
https://www.w3schools.com/tags/att_global_title.asp